crates/rpc/rpc/src/eth/signer.rs

```diff
@@ -44,8 +48,24 @@ impl EthSigner for DevSigner {
         self.accounts.contains_key(addr)
     }
 
-    async fn sign(&self, _address: Address, _message: &[u8]) -> Result<Signature> {
-        todo!()
+    async fn sign(&self, address: Address, message: &[u8]) -> Result<Signature> {
+        let secp = Secp256k1::new();
+        let secret =
+            self.accounts.get(&address).ok_or(RpcError::Custom("No account".to_string()))?;
```

Comment:
we need additional error types here:

this is what geth returns if no account found:

https://github.com/ethereum/go-ethereum/blob/ee530c0d5aa70d2c00ab5691a89ab431b73f8165/accounts/errors.go#L26-L26

Reply:
Could you go a bit more into detail on where should I add those error types? Do you mean RPCErrors or EthApi errors? Thanks :)

Reply:
we want these as additional variants in `EthApiError`, like `EthApiError::AccountNotFound`